src/wrapper.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * @author acvetkov@yandex-team.ru
+ * @overview
+ */
+
+'use strict';
+
+import {get} from './object';
+
+/**
+ * chrome api object
+ * @see https://developer.chrome.com/extensions/api_index
+ */
+const chromeApi = getChromeApi();
+
+/**
+ * Create getters for async methods
+ * @param {Object} obj
+ * @param {String} namespace
+ * @param {Array<String>} asyncMethods
+ * @param {Function} Thenable
+ * @returns {Object}
+ */
+export function wrapAsyncMethods(obj, namespace, asyncMethods = [], Thenable) {
+    return asyncMethods.reduce((result, method) => {
+        return appendAsyncMethod(obj, namespace, method, Thenable);
+    }, obj);
+}
+
+/**
+ * Wrap sync methods to promise
+ * @param {Object} obj
+ * @param {String} namespace
+ * @param {Array<String>} syncMethods
+ * @param {Function} Thenable
+ * @returns {*}
+ */
+export function wrapSyncMethods(obj, namespace, syncMethods = [], Thenable) {
+    return syncMethods.reduce((result, method) => {
+        return appendSyncMethod(obj, namespace, method, Thenable);
+    }, obj);
+}
+
+/**
+ * Get chrome api object
+ * @returns {chrome}
+ */
+function getChromeApi() {
+    if (typeof window !== 'undefined' && window.chrome) {
+        return window.chrome;
+    }
+    if (typeof global !== 'undefined' && global.chrome) {
```

Comment:
Reason for this template

`window` is global object in browser enviroment, 
`global` is global object on node platform.

But if you test your apps on node, you can have `window` polifyl, but it is not global object.

Where can be chrome object?

```js
// browser
window.chrome;
```

window is not global on node
```js
// nodejs
global.chrome; // ?
global.window.chrome; // ?
```